src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/live_event.py

```diff
@@ -15,3 +15,9 @@ def create(client, resource_group_name, account_name, name, streaming_protocol,
 
     return client.create(resource_group_name=resource_group_name, account_name=account_name,
                          live_event_name=name, parameters=live_event)
+
+
+def start(client, resource_group_name, account_name, name):
+
+    return client.start(resource_group_name=resource_group_name, account_name=account_name,
```

Comment:
There is no need to create a custom command here. You will see the sdk call is the same as the wrapper function header.